api/src/main/java/io/strimzi/api/kafka/Cmd.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka;
+
+import java.util.Objects;
+
+public class Cmd {
```

Comment:
Are you sure the `api` is the best place for this? Would it be better in operator common if it needs to be shared or something like that?